### PR TITLE
Compute event hashes in separate thread [Verifiable events part 4]

### PR DESCRIPTION
**Pull request by m4ksio**
As noted in previous PR, computing hashes can be CPU intensive, hence similar approach as with block commitments - separate comiter whish shouldn't block execution of further collections

### CODE REVIEW

File: engine/execution/computation/computer/computer.go [hunk 1]
```diff
@@ -377,3 +397,23 @@ func (bc *blockCommitter) Run() {
 func (bc *blockCommitter) Commit(view state.View) {
 	bc.views <- view
 }
+
+type evensHasher struct {
```
Comment (yhassanzadeh13) [inline engine/execution/computation/computer/computer.go:401]:
```suggestion
type eventHasher struct {
```
shouldn't it be `eventHasher`?

File: engine/execution/computation/computer/computer.go [hunk 2]
```diff
@@ -155,20 +155,38 @@ func (e *blockComputer) executeBlock(
...
+		},
+	}
+
 	go func() {
 		bc.Run()
 		wg.Done()
 	}()
 
+	go func() {
+		eh.Run()
```
Comment (yhassanzadeh13) [inline engine/execution/computation/computer/computer.go:176]: Does it make sense to make the `eventHasher` a `ReadyDoneAware` implementation? 
https://github.com/onflow/flow-go/blob/f861f317233ddec2dd49bf19f08928be0b2a9775/module/common.go#L3-L7

**Review by yhassanzadeh13 — APPROVED** [review]
LGTM

**Review by ramtinms — APPROVED** [review]
👏

File: engine/execution/computation/computer/computer.go (hunk 1 above)
Comment (m4ksio) [reply engine/execution/computation/computer/computer.go:401]: Of course! Good catch!

**Pull request merged**